code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -577,6 +577,25 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/plane_master)
 	plane = GHOST_PLANE
 	render_relay_planes = list(RENDER_PLANE_NON_GAME)
 
+/atom/movable/screen/plane_master/ghost/Initialize(mapload, datum/plane_master_group/home, offset)
+	. = ..()
+
+	var/mob/hudmod = home.our_hud.mymob
```

Comment:
just uh, do this 
```suggestion
	var/mob/hudmod = home?.our_hud?.mymob
```

Reply:
Is this really a better alternative to whitelisting it from the test? We *do* want cases of creating this without the proper arguments to error

Reply:
thinking on it further, we should blacklist all plane masters, since an initialize with no passed in plane master group will immediately qdel

that said, this is currently required regardless, because `/datum/unit_test/plane_double_transform` creates a plane master group with no hud, which is why tests fail currently. In retrospect, that shouldn't be required. but it is rn